RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -0,0 +1,427 @@
+using Microsoft.AspNetCore.Http;
+using Neo.IO.Caching;
+using Neo.IO.Data.LevelDB;
+using Neo.IO.Json;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence.LevelDB;
+using Neo.Wallets;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Neo.Ledger;
+using Neo.Persistence;
+using Snapshot = Neo.Persistence.Snapshot;
+
+namespace Neo.Plugins
+{
+    public class RpcSystemAssetTrackerPlugin : Plugin, IPersistencePlugin, IRpcPlugin
+    {
+        private const byte SystemAssetUnspentCoinsPrefix = 0xfb;
+        private const byte SystemAssetSpentUnclaimedCoinsPrefix = 0xfc;
+        private DB _db;
+        private DataCache<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs> _userUnspentCoins;
+        private bool _shouldTrackUnclaimed;
+        private DataCache<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs> _userSpentUnclaimedCoins;
+        private WriteBatch _writeBatch;
+        private int _rpcMaxUnspents;
+        private uint _lastPersistedBlock;
+        private bool _shouldPersistBlock;
+        private Neo.IO.Data.LevelDB.Snapshot _levelDbSnapshot;
+
+        public override void Configure()
+        {
+            if (_db == null)
```

Comment:
If is not `null` we shoud be able to change it, because we can change the path in the config file.

Reply:
Changing the file after syncing the chain would have to restart the plugin currently; it also isn’t very user friendly to do this right now, as it would have to reprocess all the blocks up to the current, which will take quite a long time and not indicate any progress anywhere.

Reply:
Well it can’t use the same database as the core is using, so it won’t collide with it. The method it uses takes a 1 byte prefix. We could have an additional prefix, but it doesn’t really seem needed right now.

Reply:
But if we allow to change the config file and refresh the plugins, is weird that this feature works only for certain properties, isn't it?

Reply:
Well I suppose I could make it support changing the database it uses, but it will need to cause it to reload data from all blocks. Maybe I'll add support for it, but it doesn't really seem like that useful of a feature for the effort.

Supporting reloading for `TrackUnclaimed` doesn't seem very useful either. If you turn it off and turn it back on it makes the claim information inconsistent.

What is probably the most useful and should be supported is reloading the `MaxReturnedUnspents`, it should be easy to support that.

Reply:
Added support for reloading `MaxReturnedUnspents`. I don't think it is worth adding support for changing the other settings.